### DISCUSSION TITLE
Potential issue with last version of Unleash?

### DISCUSSION THREAD

**Discussion by petko5kov** [Unleash/unleash#6033] [General]
Using Unleash in Angular 12 project. Install and build and got this error with the last version of Unleash:

```
Error: node_modules/unleash-proxy-client/build/cjs/index.d.ts:111:10 - error TS2304: Cannot find name 'type'.

111 export { type IStorageProvider, LocalStorageProvider, InMemoryStorageProvider };
```

**Comment by andreas-unleash** [comment]
Hi @petko5kov , thanks for bringing this to our attention. I looked into it a bit and the problem is a typescript version mismatch.  Angular 12 uses typescript lower that 4.4.0 and we use version ^4.4.3.  This is not an issue with latest version of Angular.  I will bring this to the team and let you know what we decide to do.

**Reply by andreas-unleash** [reply]
After talking with the team, we will no longer  support Angular 12.  (it hasn't received any security updates for the past 3 years).

**Reply by petko5kov** [reply]
Thanks!
